### PR TITLE
Made the site super responsive

### PR DESCRIPTION
**Pull request by hugmanrique**
Before, we were depending in px widths. Edited css to have two columns of servers when the device-width is larger than 1550px and one column if you are in a tablet or mobile.

- Center the category titles
- The only things that needs changing is the main graph width
- No more unneeded scrollbars
- Removed some useless CSS

### CODE REVIEW

**Comment by hugmanrique** [comment]
Some screenshots:

## Big screen - 1920px
Big screen 1920px
## Medium screen - ~600px
Medium screen

**Comment by hugmanrique** [comment]
@Cryptkeeper

**Comment by Cryptkeeper** [comment]
What happened to the Y axis labels on the 24 hour graph?

**Comment by hugmanrique** [comment]
I think it's a Flot.js issue. Will try to fix it when I arrive from vacation.

**Comment by Cryptkeeper** [comment]
Any update on this?

**Comment by hugmanrique** [comment]
I will arrive the 15th. Until then, I won't have a computer to review it

**Comment by hugmanrique** [comment]
Fixed it, the problem was the numbers were hidden because the `#big-graph` didn't have a margin

**Comment by hugmanrique** [comment]
@Cryptkeeper Any update from you?

**Comment by ThorinDev** [comment]
👍  @Cryptkeeper could we get this merged?
😄

**Comment by hugmanrique** [comment]
We will still need to fix the flot.is width issue, but could be fairly easy to fix with jQuery and .width()

**Comment by Cryptkeeper** [comment]
I'll do testing on this tonight and then merge if all goes well 😄

File: config.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		"pingAll": 3000,
 		"connectTimeout": 2500
 	},
-	"logToDatabase": false,
+	"logToDatabase": true,
```
Comment (Cryptkeeper) [inline config.json]: Can you revert this change? I'll merge it after that.

Comment (hugmanrique) [inline config.json]: Done! I think you can merge now 😎

**Comment by Cryptkeeper** [comment]
The chart issue has proved to be problematic. Can you look into fixing that and I'll re-merge after that?

**Comment by hugmanrique** [comment]
I will do it when I arrive home. It should be fairly easy with jQuery .width()

**Pull request merged**